src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ConfirmDetailsAndSendViewModelMapperTests - Copy.cs

```diff
@@ -0,0 +1,250 @@
+        }
+
+        [Test]
+        public async Task WhenFirstNameIsChanged()
+        {
+            fixture.source.FirstName = "Abc";
+
+            var result = await fixture.Map();
+
+            Assert.AreNotEqual(fixture.source.FirstName, fixture._apprenticeshipResponse.FirstName);
+            Assert.AreEqual(fixture.source.FirstName, result.FirstName);
+            Assert.AreEqual(fixture._apprenticeshipResponse.FirstName, result.OriginalApprenticeship.FirstName);
+        }
+
+        [Test]
+        public async Task WhenLastNameIsChanged()
+        {
+            fixture.source.LastName = "Abc";
+
+            var result = await fixture.Map();
+
+            Assert.AreNotEqual(fixture.source.LastName, fixture._apprenticeshipResponse.LastName);
+            Assert.AreEqual(fixture.source.LastName, result.LastName);
+            Assert.AreEqual(fixture._apprenticeshipResponse.LastName, result.OriginalApprenticeship.LastName);
+        }
+
+        [Test]
+        public async Task WhenDobIsChanged()
+        {
+            fixture.source.DateOfBirth = new CommitmentsV2.Shared.Models.DateModel(new DateTime(2000,12,31));
+
+            var result = await fixture.Map();
+
+            Assert.AreNotEqual(fixture.source.DateOfBirth.Day, fixture._apprenticeshipResponse.DateOfBirth.Day);
+            Assert.AreEqual(fixture.source.DateOfBirth.Date, result.DateOfBirth);
+            Assert.AreEqual(fixture._apprenticeshipResponse.DateOfBirth, result.OriginalApprenticeship.DateOfBirth);
+        }
+
+        [Test]
+        public async Task WhenStartDateIsChanged()
+        {
+            var newStartDate = fixture._apprenticeshipResponse.StartDate.AddMonths(-1);
+            fixture.source.StartDate = new CommitmentsV2.Shared.Models.MonthYearModel(newStartDate.Month.ToString() + newStartDate.Year);
+
+            var result = await fixture.Map();
+
+            Assert.AreNotEqual(fixture.source.StartDate.Date, fixture._apprenticeshipResponse.StartDate);
+            Assert.AreEqual(fixture.source.StartDate.Date, result.StartDate);
+            Assert.AreEqual(fixture._apprenticeshipResponse.StartDate, result.OriginalApprenticeship.StartDate);
+        }
+
+        [Test]
+        public async Task WhenEndDateIsChanged()
+        {
+            var newEndDate = fixture._apprenticeshipResponse.EndDate.AddMonths(-1);
+            fixture.source.EndDate = new CommitmentsV2.Shared.Models.MonthYearModel(newEndDate.Month.ToString() + newEndDate.Year);
+
+            var result = await fixture.Map();
+
+            Assert.AreNotEqual(fixture.source.EndDate.Date, fixture._apprenticeshipResponse.EndDate);
+            Assert.AreEqual(fixture.source.EndDate.Date, result.EndDate);
+            Assert.AreEqual(fixture._apprenticeshipResponse.EndDate, result.OriginalApprenticeship.EndDate);
+        }
+
+        [Test]
+        public async Task WhenCourseIsChanged()
+        {
+            fixture.source.CourseCode = "Abc";
```

Comment:
"Abc" used for coursecode,firstname, lastname, employerref .. can we change  to different related strings please